CrazyCanvas/Source/ECS/Systems/GUI/HUDSystem.cpp

```diff
@@ -17,6 +17,7 @@
 #include "Application/API/Events/EventQueue.h"
 
 #include "Lobby/PlayerManagerClient.h"
+#include "..\..\..\..\Include\ECS\Systems\GUI\HUDSystem.h"
```

Comment:
I think this can be accessed in an easier way. I think you can go straight to `ECS\Systems\GUI\HUDSystem.h`

Reply:
yes, I have no idea why this has changed xD